src/core/annotation.js

```diff
@@ -1110,6 +1109,61 @@ class TextAnnotation extends MarkupAnnotation {
   }
 }
 
+class FreeTextAnnotation extends Annotation {
```

Comment:
Could you move this back to the original place in the file? That will make the diff smaller and easier to review.